constellation-app/constellation
# Documentation/updating documentation in surge 5

### Prerequisites


    for the "CI / Unit Tests") to complete first. Failed Unit tests can be 
    debugged by adding the label "verbose logging" to the GitHub PR.

### Description of the Change


Updated the following help pages:

- Scatter plot view
- Timeline View

### Alternate Designs


N/A

### Why Should This Be In Core?


N/A

### Benefits


Updated documentation

### Possible Drawbacks


None

### Verification Process



1. Open Consty
2. Enter the following in the search bar and observe the changes to the help pages:

- scatter-plot-view
- timeline-view

### Applicable Issues


https://github.com/constellation-app/constellation/issues/340
